### DISCUSSION TITLE
CSS for Unordered Lists and Nested Lists

### DISCUSSION THREAD

**Discussion by MatthewMcD** [onweru/newsroom#36] [Ideas]
I noticed that the UL and nested UL don't display icons or indentation:
https://themes.gohugo.io//theme/newsroom/post/markdown-syntax/

Is this a design intent?

**Comment by MatthewMcD** [comment]
I started fixing this for my site by adding a custom css file. I added a `[param]` for `custom_css` since this is how I have done it on other Hugo sites. I haven't used SASS before.

Would you like a PR that adds this to the `head.html` partial?

**Reply by onweru** [reply]
@MatthewMcD, yes please. sorry I couldn't respond sooner.

**Comment by janw76** [comment]
I would love to change that too, but I am not sure how to. @MatthewMcD any chance you could do the PR?

**Reply by onweru** [reply]
@janw76, 

I have updated the theme to style the unordered lists.

**Reply by janw76** [reply]
Oh wow, thanks so much!
